data_classes.py

```diff
@@ -0,0 +1,139 @@
+from dataclasses import dataclass, field
+from datetime import datetime
+import configs.constants as constants
+import json
+import logging
+import logging.config
+
+with open("configs/log_config.json", "r", encoding="UTF-8") as stream:
+    config = json.load(stream)
+logging.config.dictConfig(config)
+logger = logging.getLogger("standard")
+
+
+@dataclass
+class TokenSale:
+    """
+    Data of given sale transaction
+    """
+
+    sort_index: int = field(init=False)
+    avax_price_in_usd: float

```

Comment:
what's the `field(init=False)` for?

Reply:
tried to do fancy sorting with dataclass but it didn't work out, so found workaround. removed now